packages/cascara/src/ui/Section/Section.module.scss

```diff
@@ -0,0 +1,15 @@
+.Section {
+  position: relative;
+  background: #fff;
+  margin: 1rem 0;
+  padding: 1em 1em;
```

Comment:
@lordpixel in the future, if we have vertical and inline values that are the same for padding (or margins) we should go ahead and just put the value once. No need to change this though. I have a feeling we are potentially going to need to tweak some of these things across Cascara in a couple months anyway so the values can come from design tokens.

Good work replicating the styes.